src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.System/Main.cs

```diff
@@ -135,7 +135,7 @@ public List<Result> Query(Query query)
         public List<Result> Query(Query query, bool delayedExecution)
         {
             List<Result> results = new List<Result>();
-            CultureInfo culture = _localizeSystemCommands ? CultureInfo.CurrentUICulture : new CultureInfo("en-US");
+            CultureInfo culture = _localizeSystemCommands ? CultureInfo.CurrentUICulture : new CultureInfo("en-US", false);
```

Comment:
same here.

Reply:
Thanks for the notice. While it wouldn't hurt, I still reverted it ;)